src/main/java/frc/robot/utilities/pid/PIDFMotor.java

```diff
@@ -1,12 +1,22 @@
 package frc.robot.utilities.pid;
 
-import com.ctre.phoenix.motorcontrol.ControlMode;
+import edu.wpi.first.util.sendable.SendableBuilder;
 
 /**
  * An interface used for motors that have a configurable PID
  */
 public interface PIDFMotor extends PIDFConfigurable {
     void setCoefs(PIDFCoefs pidfCoefs);
 
-    void tunePIDF(ControlMode controlMode);
+    void set(double output);
```

Comment:
why can we remove tunePIDF exactly?

Reply:
because we have `setSetpoint()` defined in `PIDConfigurable`

Reply:
setSetpoint does not actually move the motor. in the pidController you need to use caculate and in talons you need to do set

Reply:
PIDFMotor doesn't use a PIDFController, it uses the Talon's onboard PID